frontend/src/metabase/visualizations/visualizations/Map.jsx

```diff
@@ -341,3 +349,27 @@ export default class Map extends Component {
     }
   }
 }
+
+const mapStateToProps = (state, props) => {
+  return {
+    isAdmin: getUserIsAdmin(state, props),
+  };
+};
+
+const CustomMapFooter = connect(mapStateToProps)(({ isAdmin }) => (
+  <CustomMapContainer>
+    <Button
+      borderless
+      iconRight="share"
+      fullWidth
+      as="a"
+      href={
+        isAdmin
+          ? "/admin/settings/maps"
+          : "https://www.metabase.com/docs/latest/configuring-metabase/custom-maps"
+      }
+    >
+      Custom Map
```

Comment:
Not translated

Reply:
![200](https://user-images.githubusercontent.com/1328979/193286898-b50afc77-7730-4717-8268-82b8898dfb99.gif)